src/GitVersionCore/Extensions/GitExtensions.cs

```diff
@@ -9,19 +9,19 @@ namespace GitVersion.Extensions
 {
     public static class GitExtensions
     {
-        public static DateTimeOffset When(this Commit commit)
+        public static DateTimeOffset When(this ICommit commit)
```

Comment:
Any reason not to provide `When` as a property on the `ICommit` interface?

Reply:
Agree, I will change that in a subsequent PR